cms/djangoapps/contentstore/management/commands/delete_course.py

```diff
@@ -21,12 +21,36 @@
 class Command(BaseCommand):
     """
     Delete a MongoDB backed course
+    Example usage:
+        $ ./manage.py cms delete_course 'course-v1:edX+DemoX+Demo_Course' --settings=devstack
+        $ ./manage.py cms delete_course 'course-v1:edX+DemoX+Demo_Course' --keep-instructors --settings=devstack
+
+    Note:
+        --keep-instructors option is added in effort to delete duplicate courses safely.
+        There happens to be courses with difference of casing in ids, for example
+        course-v1:DartmouthX+DART.ENGL.01.X+2016_T1 is a duplicate of course-v1:DartmouthX+DART.ENGL.01.x+2016_T1
+        (Note the differene in 'x' of course number). These two are independent courses in MongoDB.
+        Current MYSQL setup is case-insensitive which essentially means there are not
+        seperate entries (in all course related mysql tables, but here we are concerned about accesses)
+        for duplicate courses.
+        This option will make us able to delete course (duplicate one) from
+        mongo while perserving course's related access data in mysql.
     """
     help = '''Delete a MongoDB backed course'''
 
     def add_arguments(self, parser):
+        """
+        Add arguments to the command parser.
+        """
         parser.add_argument('course_key', help="ID of the course to delete.")
 
+        parser.add_argument(
+            '--keep-instructors',
+            action='store_true',
+            default=False,
+            help='Do not revoke users and groups permissions from course',
```

Comment:
Is it not making sense? Keeping instructors essentially means not revoking permissions. Happy to reword it on you suggestions.